src/main/resources/log4j.properties

```diff
@@ -0,0 +1,28 @@
+#
+# @(#) log4j.properties Copyright (c) 2019 Jala Foundation.
```

Comment:
the file with extension properties does not have to have copyright

Reply:
Fixed.